client/src/assets/config/config.template

```diff
@@ -1,4 +1,5 @@
 {
     "apiEndpoint": "API-ENDPOINT",
-    "googleAnalyticsId": "GOOGLE-ANALYTICS-ID"
+    "googleAnalyticsId": "GOOGLE-ANALYTICS-ID",
+    "identityResolverEnabled": false
```

Comment:
Can we change this to 
`"identityResolverEnabled": IDENTITY-RESOLVER-ENABLED`
as we do a string replace on deploy, which is harder with `false`